package.json

```diff
@@ -78,59 +81,58 @@
   },
   "devDependencies": {
     "@ember/optional-features": "^2.0.0",
-    "@ember/test-helpers": "^2.4.1",
-    "@embroider/test-setup": "^0.39.0",
-    "@glimmer/component": "^1.0.1",
-    "@glimmer/tracking": "^1.0.1",
+    "@ember/test-helpers": "^2.6.0",
+    "@embroider/test-setup": "^0.49.0",
+    "@glimmer/component": "^1.0.4",
+    "@glimmer/tracking": "^1.0.4",
     "@types/ember": "^3.16.0",
-    "@types/ember-qunit": "^3.4.9",
-    "@types/ember__test-helpers": "^2.0.0",
-    "@types/qunit": "^2.9.1",
+    "@types/ember-qunit": "^3.4.15",
+    "@types/ember__test-helpers": "^2.6.0",
+    "@types/qunit": "^2.11.3",
     "@types/rsvp": "^4.0.3",
     "@typescript-eslint/eslint-plugin": "^4.0.0",
     "@typescript-eslint/parser": "^4.0.0",
     "babel-eslint": "^10.1.0",
     "broccoli-asset-rev": "^3.0.0",
-    "ember-cli": "^3.20.0",
+    "ember-cli": "^4.0.1",
     "ember-cli-dependency-checker": "^3.2.0",
     "ember-cli-github-pages": "^0.2.2",
-    "ember-cli-htmlbars": "^5.2.0",
-    "ember-cli-inject-live-reload": "^2.0.2",
-    "ember-cli-release": "^0.2.9",
+    "ember-cli-htmlbars": "^6.0.1",
+    "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sass": "^10.0.1",
     "ember-cli-sri": "^2.1.1",
+    "ember-cli-terser": "^4.0.2",
     "ember-cli-typescript-blueprints": "^3.0.0",
-    "ember-cli-uglify": "^3.0.0",
     "ember-disable-prototype-extensions": "^1.1.3",
     "ember-export-application-global": "^2.0.1",
-    "ember-load-initializers": "^2.1.1",
-    "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-qunit": "^5.1.4",
-    "ember-resolver": "^8.0.2",
+    "ember-load-initializers": "^2.1.2",
+    "ember-qunit": "^5.1.5",
+    "ember-resolver": "^8.0.3",
     "ember-sinon": "^5.0.0",
-    "ember-source": "^3.20.4",
+    "ember-source": "^4.0.1",
     "ember-source-channel-url": "^3.0.0",
-    "ember-template-lint": "^3.0.0",
+    "ember-template-lint": "^3.15.0",
     "ember-truth-helpers": "^3.0.0",
-    "ember-try": "^1.4.0",
-    "eslint": "^7.7.0",
-    "eslint-config-prettier": "^8.0.0",
-    "eslint-plugin-ember": "^10.0.0",
+    "ember-try": "^2.0.0",
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-ember": "^10.5.8",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^3.1.4",
-    "express": "^4.17.1",
+    "eslint-plugin-prettier": "^4.0.0",
```

Comment:
I wonder if we should remove this in favor of using `prettier --check .`, which is [recommended](https://prettier.io/docs/en/integrating-with-linters.html#notes).

Reply:
@scalvert interesting! I think we may do so, this comes from default ember-cli blueprint, so not sure if it worth to diverge here

Reply:
Ya let's leave it for now, and we can work on updating the default blueprint.